DataFormats/PatCandidates/interface/TauPFEssential.h

```diff
@@ -48,6 +48,12 @@ struct TauPFEssential {
   reco::VertexRef sv_;
   Point svPos_;
   CovMatrix svCov_;
+  float ip3d_;
+  float ip3d_error_;
+  float ip3d_Sig_;
```

Comment:
I was talking about ip3d_Sig_.
There is no other line in this PR that modifies or uses this variable.
If you are planning to fill it later, it has to be at least initialized to some default values, so that we do not store random floats (this is what happens with uninitialized data members).